src/nwb_project_analytics/dandistats.py

```diff
@@ -242,7 +243,7 @@ def plot_number_of_dandisets_by_date(dandistats_df: pd.DataFrame,
                 linewidth=0.5)
         for label in ax.get_xticklabels(which='major'):
             label.set(rotation=30, horizontalalignment='right')
-        ax.set_ylabel('# of NWB Dandisets')
+        ax.set_ylabel('# of NWB dandisets')
```

Comment:
```suggestion
        ax.set_ylabel('# of NWB Dandisets')
```

Dandiset is correct: https://github.com/dandi/handbook#dandi-style-guidelines